src/main/kotlin/com/hedvig/underwriter/model/QuoteData.kt

```diff
@@ -32,6 +32,14 @@ sealed class QuoteData {
             is DanishAccidentData -> ProductType.ACCIDENT
         }
     }
+
+    fun market(): Market {
+        return when (this) {
+            is SwedishHouseData, is SwedishApartmentData -> Market.SWEDEN
+            is DanishAccidentData, is DanishTravelData, is DanishHomeContentsData -> Market.DENMARK
+            is NorwegianTravelData, is NorwegianHomeContentsData -> Market.NORWAY
+        }
+    }
```

Comment:
This can be a `val` to highlight that it's a simple deduction and not significant work.

```suggestion
    val market: Market get() = when (this) {
        is SwedishHouseData, is SwedishApartmentData -> Market.SWEDEN
        is DanishAccidentData, is DanishTravelData, is DanishHomeContentsData -> Market.DENMARK
        is NorwegianTravelData, is NorwegianHomeContentsData -> Market.NORWAY
    }
}
```

Reply:
Thanks for the feedback. Will update to this.

I really appreciates feedback like this. My Kotlin skills is still very immature and this helps me to progress me learning it faster. <3!